src/main/java/com/squareup/javapoet/TypeSpec.java

```diff
@@ -234,6 +242,10 @@ void emit(CodeWriter codeWriter, String enumName, Set<Modifier> implicitModifier
     }
   }
 
+  public boolean isAnnotationDeclaration() {
```

Comment:
We might as well just make it public. This API doesn't try very hard to encapsulate things.